tests/test_ops/test_onnx.py

```diff
@@ -23,6 +23,66 @@ def forward(self, *args, **kwargs):
         return self.wrapped_function(*args, **kwargs)
 
 
+@pytest.mark.parametrize('mode', ['bilinear', 'nearest'])
+@pytest.mark.parametrize('padding_mode', ['zeros', 'border', 'reflection'])
+@pytest.mark.parametrize('align_corners', [True, False])
+def test_grid_sample(mode, padding_mode, align_corners):
+    from mmcv.onnx.symbolic import register_extra_symbolics
+    opset_version = 11
+    register_extra_symbolics(opset_version)
+
+    from mmcv.ops import get_onnxruntime_op_path
+    ort_custom_op_path = get_onnxruntime_op_path()
+    if not os.path.exists(ort_custom_op_path):
+        pytest.skip('nms for onnxruntime is not compiled.')
```

Comment:
Custom ops for onnxruntime are not compiled.

Reply:
> Custom ops for onnxruntime are not compiled.

ok